### PR TITLE
Rename Status as Consistent

### PR DESCRIPTION
**Pull request by july2993**
### What problem does this PR solve? 
see https://github.com/pingcap/docs-cn/pull/2224#discussion_r376911866

### What is changed and how it works?
Rename Status as Consistent

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: drainer/checkpoint/checkpoint.go [hunk 1]
```diff
@@ -19,14 +19,6 @@ import (
 	"go.uber.org/zap"
 )
 
-const (
-	// StatusConsistent means server quit normally, data <= ts is synced to downstream
-	StatusConsistent int = 0
```
Comment (GregoryIan) [inline drainer/checkpoint/checkpoint.go]: `Status` is more flexible and scalable. I think It's better if we map status to readable words.

Comment (july2993) [reply drainer/checkpoint/checkpoint.go]: what about keep the origin Status and add a filed `StatusMessage` in the checkpoint ?

File: drainer/checkpoint/checkpoint.go [hunk 2]
```diff
@@ -39,13 +31,13 @@ type CheckPoint interface {
...
-	Save(commitTS int64, slaveTS int64, status int) error
+	Save(commitTS int64, slaveTS int64, consistent bool) error
 
 	// TS gets checkpoint commit timestamp.
 	TS() int64
 
-	// Status return the status saved.
-	Status() int
+	// Consistent return the Consistent status saved.
+	Consistent() bool
```
Comment (suzaku) [inline drainer/checkpoint/checkpoint.go]: `IsConsistent`?

**Comment by GregoryIan** [comment]
LGTM

File: drainer/checkpoint/checkpoint.go (hunk 2 above)
Comment (july2993) [reply drainer/checkpoint/checkpoint.go]: change  by https://github.com/pingcap/tidb-binlog/pull/895/commits/d0d404718584604fe5428d7d5c276129f9c36dea

**Pull request merged**